supabase/migrations/20240418124604_add_created_at_column_audit_log_table.sql

```diff
@@ -0,0 +1,3 @@
+alter table "public"."audit_log" add column "created_at" timestamp with time zone default current_timestamp;
+update "public"."audit_log" set created_at = current_timestamp;
+alter table "public"."audit_log" alter column "created_at" set not null;
```

Comment:
Timestamp in UTC will be automatically generated when a row is inserted. I've set all existing rows to time stamp when migration happens.